lua/telescope/config.lua

```diff
@@ -77,44 +89,49 @@ function config.set_defaults(defaults)
     end
   end
 
-  set("sorting_strategy", "descending", [[
+  for key, info in pairs(tele_defaults) do
+    set(key, info[1], info[2])
+  end
+end
+
+telescope_defaults["sorting_strategy"] = { "descending", [[
```

Comment:
why did you make this change? I don't remember talking about it (just looking for context)

Reply:
You're right, we didn't discuss this bit.

This was part of the switch I made to allow `config.set_defaults` to be passed in the Telescope defaults as well as the users defaults. If nothing is passed in for Telescope defaults, then we use the table `telescope_defaults`, which contains all of the previous defaults.

This makes it easier to test the inheritance in `layout_strategies_spec.lua` [here](https://github.com/nvim-telescope/telescope.nvim/blob/9699e68379996e0167c9e97feaa3601463eb358a/lua/tests/automated/layout_strategies_spec.lua#L41).
With this set up we can use different defaults for "testing" compared to "production", which makes it a bit easier to write (and understand) the tests.